packages/video_player/video_player/AUTHORS

```diff
@@ -65,3 +65,5 @@ Anton Borries <mail@antonborri.es>
 Alex Li <google@alexv525.com>
 Rahul Raj <64.rahulraj@gmail.com>
 Koen Van Looveren <vanlooverenkoen.dev@gmail.com>
+Gabriel Motelevicz Okura <gabriellevicz@gmail.com>
+Idowu Tomiwa <tommydprogrammer@gmail.com>
```

Comment:
Why are you adding a second name here?

Reply:
As you said before, this PR was addressing the same issue as #3510. 

So i talked with the author of the other PR and we agreed to put his name in this PR too.